### ISSUE TITLE
NoUISlider breaking completely in angular 13 version.

### ISSUE THREAD

**Issue by ridhi58** [leongersen/noUiSlider#1222]
Hey,

I have upgraded my project to angular 13 . NoUiSlider was working in old angular versions but with angular 13 it is completely breaking .
I just want to confirm if it is not compatible with angular 13 .

**Comment by ridhi58** [comment]
for your reference . the slider breaks like this in newer version. 
![image](https://user-images.githubusercontent.com/67142751/186095985-1f54845c-69be-4774-b690-5bce052f0141.png)

**Comment by leongersen** [comment]
I'm assuming you've updated to [15.5.0 or later](https://github.com/leongersen/noUiSlider/blob/master/CHANGELOG.MD#1550-2021-09-01). Your custom styles will need updating. 

[These changes in the default stylesheet](https://github.com/leongersen/noUiSlider/commit/a1f8f17fa7a2a2ea81a8f9a4fd7c4692ebe16927) need to be incorporated in your overriding styles.

See #1167 for more context.

**Comment by ridhi58** [comment]
i am using  "nouislider": "^15.6.0", version.

**Comment by ridhi58** [comment]
is there a way , i dont have to manually override the styles everytime there are some changes with changing the verison?

**Comment by leongersen** [comment]
No, if you override the included styles you'll have to keep them compatible. The break change here is unfortunate, and I would've avoided it in hindsight. Again, see #1167 for context.
